block/stream/output/consensus_service.proto

```diff
@@ -78,90 +78,90 @@ message SubmitMessageOutput {
      * For all current transactions, this value SHALL be
      * `WITH_MESSAGE_DIGEST_AND_PAYER`.
      */
-    RunningHashVersion topic_running_hash_version = 1;
+    SubmitMessageOutputRunningHashVersion topic_running_hash_version = 1;
+}
 
+/**
+ * A version of the topic running hash.
+ *
+ * The inputs to the topic running hash have changed over time.
+ * This is tracked in earlier record streams as an integer. For the
+ * block stream we chose to use an enumeration for both efficiency
+ * and clarity. Placing the most recent, and most common/highest
+ * volume, version as `0` reduces the serialized size of this message
+ * by not serializing that default value.
+ *
+ * <hr style="margin: 0.2em 5em 0.2em 5em; height: 0.5em; border-style: solid none solid none; border-width: 2px;"/>
+ *
+ * The topic running hash is a 48-byte value that is the output
+ * of a SHA-384 digest with input data determined by the value of
+ * the `topic_running_hash_version` field.<br/>
+ * All new transactions SHALL use `topic_running_hash_version`
+ * `WITH_MESSAGE_DIGEST_AND_PAYER`.<br/>
+ */
+enum SubmitMessageOutputRunningHashVersion {
```

Comment:
This is general, it can just be `RunningHashVersion`, as it applies across the system, not just for one message.  I was planning to extract this to top level in the next update regardless.